less/hotgraphic.less

```diff
@@ -63,4 +63,15 @@
   &__tile.is-visited .icon {
     .icon-tick;
   }
+
+}
+
+.hotgraphic__pin-tooltip.is-static {
+  --adapt-tooltip-distance: 0;
+  --adapt-tooltip-arrow: false;
+
+  .tooltip__body {
+    color: black;
+    background-color: transparent;
+  }
```

Comment:
Without this it would look like standard tooltips:
![image](https://github.com/adaptlearning/adapt-contrib-hotgraphic/assets/7974663/a9f466b9-6241-42ac-8803-6b4f53b50d4a)

Reply:
@oliverfoster I've moved these styles up to the pin section, so this is out of date.

Otherwise, I like that these styles are included here to show themers how to easily customize the appearance. I'm uncertain if we should just have the styles default to the normal tooltip styles. I could go either way, as they'll probably be modified in a custom theme anyway. Great work, though!

Reply:
Thanks